### DISCUSSION TITLE
Librum Android client

### DISCUSSION THREAD

**Discussion by demindx** [Librum-Reader/Librum#126] [General]
Will librum get an android client in the future?

**Comment by DavidLazarescu** [comment]
Hey, yes. We are already working on porting Librum to Android.

**Reply by tbleiker** [reply]
I'am eagerly waiting for the android app to test... 😀 When porting, do you take care to optimize the app for e-readers?

**Reply by DavidLazarescu** [reply]
What exactly do you mean with optimising for e-readers? Are you referring to hardware readers like kindle?

**Reply by tbleiker** [reply]
I have a Boox Page from Onyx. It runs Android. However, since its an eInk display (no color) a few things have to be considered to make it usable on such a device.

**Reply by DavidLazarescu** [reply]
We are currently working on the default Android application. We will make sure to cover special cases such as Boox Pages later on.

**Reply by Dr-42** [reply]
How far along are we? Also how can build till the current progress? as the platform/android branch doesn't have build instructions for android

**Comment by DavidLazarescu** [comment]
We made some progress but still need to make some major changes to the rendering process for mobile to allow smooth movement. I am actively working on this but dont have much free time atm. so development is a little slow.

**Reply by Dr-42** [reply]
Thank you so much for keeping us updated. Great to have developers like you!
